### PR TITLE
Feature/add action-yamllint

### PR DESCRIPTION
**Pull request by vk26**
- [x] This PR has been added to [CHANGELOG.md](https://github.com/dotenv-linter/dotenv-linter/blob/master/CHANGELOG.md) (at the top of the list);
- [x] Tests for the changes have been added (for bug fixes / features);
- [x] Docs have been added / updated (for bug fixes / features).

### CODE REVIEW

File: .github/workflows/yamllint.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name: yamllint
+on: [pull_request]
+
+env:
+  CARGO_TERM_COLOR: always
```
Comment (mgrachev) [inline .github/workflows/yamllint.yml:5]:
```suggestion
```
You don't run `cargo`, so please remove these lines.

File: .github/workflows/yamllint.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  yamllint:
+    name: check / yamllint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: yamllint
+        uses: reviewdog/action-yamllint@v1
+        with:
+          github_token: ${{ secrets.github_token }}
+          reporter: github-pr-review
```
Comment (mgrachev) [inline .github/workflows/yamllint.yml:17]: Please add the `fail_on_error: true` option too.
https://github.com/reviewdog/action-yamllint#fail_on_error

**Review by mgrachev — COMMENTED** [review]
@vk26 👋 Thank you for your contribution! 🚀

I have left some comments. Please look at them 👀

**Comment by mgrachev** [comment]
Also, I see some comments from yamllint. Please fix them.

**Comment by vk26** [comment]
@mgrachev Hi! I fixed it. I had to use key "on" with quotes following [yamllint rule about truthy](https://yamllint.readthedocs.io/en/stable/rules.html?highlight=truthy%20value%20should%20be#module-yamllint.rules.truthy)

**Comment by mgrachev** [comment]
> @mgrachev Hi! I fixed it. I had to use key "on" with quotes following [yamllint rule about truthy](https://yamllint.readthedocs.io/en/stable/rules.html?highlight=truthy%20value%20should%20be#module-yamllint.rules.truthy)

Can we skip this rule using the [yamllint_flags](https://github.com/reviewdog/action-yamllint#yamllint_flags) option?

**Comment by vk26** [comment]
@mgrachev Hi! I set in yamllint_flags skipping truthy-rule.

**Review by mgrachev — APPROVED** [review]

**Comment by mgrachev** [comment]
@vk26 Thank you! ❤️

**Pull request merged**